### PR TITLE
Refactored readme documentation for Gaudi examples

### PR DESCRIPTION
**Pull request by Supriya-Krishnamurthi**
This PR contains the following changes suggested by Habana team:

1.  Renaming of folder name "HPU" to "GAUDI" under interactive API Folder
2.  Added new Readme file shared by Henry under GAUDI folder that contains HPU examples
3.  Updated Readme file for all 3 HPU examples based on the inputs received during our discussion with Habana Team

### CODE REVIEW

File: openfl-tutorials/interactive_api/GAUDI/PyTorch_Kvasir_UNet/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# PyTorch_Kvasir_UNet
+
+## **Habana Tutorials**
+#### The name of the file/example that contain HPU adaptations start with "HPU".
```
Comment (bharagha) [inline openfl-tutorials/interactive_api/GAUDI/PyTorch_Kvasir_UNet/README.md]: Given the name change, check HPU here.
Also is the comment valid?

File: openfl-tutorials/interactive_api/GAUDI/PyTorch_Kvasir_UNet/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+ **Note:** By default these experiments utilize 1 HPU device
+ <br/>
+
+ ## **I. Habana Setup**
+
+ Please follow the instructions provided in the [Gaudi Installation Guide](https://docs.habana.ai/en/latest/Installation_Guide/index.html) to set up the environment including the ```$PYTHON``` environment variable.
+ <br/>
+ 
+ This example was tested on AWS DL1 EC2 instance created by following the instructions mentioned [here](https://docs.habana.ai/en/latest/AWS_EC2_DL1_and_PyTorch_Quick_Start/AWS_EC2_DL1_and_PyTorch_Quick_Start.html) .
```
Comment (bharagha) [inline openfl-tutorials/interactive_api/GAUDI/PyTorch_Kvasir_UNet/README.md]: Add a note: :"The example can be executed on any Gaudi configuration, DL1 on AWS or on an on-prem configuration. Currently, the validation has only been done on DL1 instance on AWS".

File: openfl-tutorials/interactive_api/GAUDI/PyTorch_Kvasir_UNet/README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+<br/>
+ 
+ ## **II. HPU Adaptations For PyTorch Examples**
+
+Please refer  [getting started with PyTorch](https://docs.habana.ai/en/latest/PyTorch/Getting_Started_with_PyTorch_and_Gaudi/Getting_Started_with_PyTorch.html) for detailed explaination of the Habana specific code modifications needed to run the model on HPU. Additionally the document provides sample PyTorch example with Habana specifc adaptations for reference, followed by set of instructions to execute it.
+<br/>
+
+### **Note**:
+
+**There is a dependency in the order of execution (moving model to HPU and intializing optimizer). The workaround is to execute this step before initializing any optimizers.**
```
Comment (bharagha) [inline openfl-tutorials/interactive_api/GAUDI/PyTorch_Kvasir_UNet/README.md]: "execute this step". Instead say, "execute the below step".

File: openfl-tutorials/interactive_api/GAUDI/PyTorch_MedMNIST_2D/README.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+![MedMNISTv2_overview](https://raw.githubusercontent.com/MedMNIST/MedMNIST/main/assets/medmnistv2.jpg)
+
+For more details, please refer to the original paper:
+**MedMNIST v2: A Large-Scale Lightweight Benchmark for 2D and 3D Biomedical Image Classification** ([arXiv](https://arxiv.org/abs/2110.14795)), and [PyPI](https://pypi.org/project/medmnist/).
+
+
+## **Habana Tutorials**
+
+## **About model and experiments**
+
```
Comment (bharagha) [inline openfl-tutorials/interactive_api/GAUDI/PyTorch_MedMNIST_2D/README.md:12]: Apply my comments from the Kvasir documentation review.

File: openfl-tutorials/interactive_api/GAUDI/PyTorch_TinyImageNet/README.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+## **Habana Tutorials**
+#### The name of the file/example that contain HPU adaptations start with "HPU". 
+For example: HPU_pytorch_tinyimagenet.ipynb placed under workspace folder contains the required HPU adaptations.
+
+ All the execution steps mention in last section (**III. How to run this tutorial**) remain same for HPU examples but as pre-requisite it needs some additional environment setup and Habana supported package installations which is explained below in **section I and II**.
+
+**Note:** By default these experiments utilize only 1 HPU device
+<br/>
+
```
Comment (bharagha) [inline openfl-tutorials/interactive_api/GAUDI/PyTorch_TinyImageNet/README.md:10]: Comments from previous readme review are applicable here. Please apply them.

**Review by ytang-habana — APPROVED** [review]

**Comment by MasterSkepticista** [comment]
Closing as PR is stale and out-of-date with recent G3 documentation.

**Pull request closed without merging**